src/redux.js

```diff
@@ -173,154 +173,166 @@ const initialState = {
   accuracyCheckExamples: [],
   accuracyCheckLabels: [],
   accuracyCheckPredictedLabels: [],
-  showPredict: false,
   testData: {},
   prediction: {},
   modelSize: undefined,
   trainedModel: undefined,
-  trainedModelDetails: {}
+  trainedModelDetails: {},
+  currentPanel: "selectDataset",
 };
 
 // Reducer
 export default function rootReducer(state = initialState, action) {
   if (action.type === SET_MODE) {
     return {
       ...state,
-      mode: action.mode
+      mode: action.mode,
     };
   }
   if (action.type === SET_SELECTED_CSV) {
     return {
       ...state,
-      csvfile: action.csvfile
+      csvfile: action.csvfile,
     };
   }
   if (action.type === SET_SELECTED_JSON) {
     return {
       ...state,
-      jsonfile: action.jsonfile
+      jsonfile: action.jsonfile,
     };
   }
   if (action.type === SET_IMPORTED_DATA) {
     return {
       ...state,
-      data: action.data
+      data: action.data,
```

Comment:
what's up with all the trailing commas? does your editor auto add them?

Reply:
My local JsPrettier settings somehow got reset; will fix.